ykcs11/mechanisms.c

```diff
@@ -706,6 +706,17 @@ CK_RV check_pvtkey_template(gen_info_t *gen, CK_MECHANISM_PTR mechanism, CK_ATTR
       break;
 
     case CKA_SENSITIVE:
+      if (*((CK_BBOOL *)templ[i].pValue) != CK_FALSE) {
```

Comment:
Logic here and in extractable should be inverted: if the value of sensitive is not equal to true -> then reject. Similarily for extractable -> if it's not false then reject.

Reply:
D'oh, thanks...
I force-pushed an updated version. Would you mind having a look?

Reply:
Sorry, one more issue I overlooked on my first review :)